src/controller/BackPostController.php

```diff
@@ -26,7 +26,7 @@ public function addPost(PostModel $postModel)
 
 		// The form is not submitted, posting the post form
 		if (count($_POST) === 0) {
-			$this->render('insert_post.twig', $_SESSION);
+			$this->render('insert_post.twig', [], [], $_SESSION, []);
```

Comment:
1- Mettre session en paramètre non optionnel et en second paramètre de ta fonction `private function render(string $templateName, array $errors = [], array $form = [], array $session = [], array $post = [])`. Finalement à quoi sert un paramètre optionnel si tu en as toujours besoin ? 
```suggestion
			$this->render('insert_post.twig', $_SESSION);
```

2- De plus as tu bien compris le concept d'un `paramètre optionel` dans une fonction ? dans ton changement tu mets encore un tableau vide à la fin de ta fonction hors ton paramètre $post est "optionnel" donc si tu ne le renseigne pas dans ton `$this->render(....)` il prendra automatiquement la valeur d'un tableau vide comme indiqué dans la déclaration de ta fonction `private function render(string $templateName, array $errors = [], array $form = [], array $session = [], array $post = [])` => `array $post = []`

Reply:
Modif ok : 
$this->render('insert_post.twig', $_SESSION);